### PR TITLE
Localization for 2019.1

### PR DESCRIPTION
**Pull request by stephenmichaelf**

### CODE REVIEW

File: make.js [hunk 1]
```diff
@@ -560,6 +564,16 @@ target.bump = function() {
         }
 
         taskJson.version.Patch = taskJson.version.Patch + 1;
+        taskLocJson.version.Patch = taskLocJson.version.Patch + 1;
```
Comment (damccorm) [inline make.js]: Can we just do `taskLocJson.version.Patch = taskJson.version.Patch` here and not do the verification? It seems like that will always be the right behavior.

Comment (stephenmichaelf) [reply make.js]: I was debating that but then that would indicate a user error from not bumping both so I wanted to be aware of it explicitly.

File: make.js [hunk 2]
```diff
@@ -560,6 +564,16 @@ target.bump = function() {
...
+
         fs.writeFileSync(taskJsonPath, JSON.stringify(taskJson, null, 4));
+        fs.writeFileSync(taskJsonPath, JSON.stringify(taskLocJson, null, 2));
```
Comment (damccorm) [inline make.js]: This should be taskLocJsonPath not taskJsonPath

File: make.js [hunk 3]
```diff
@@ -560,6 +564,16 @@ target.bump = function() {
...
+
+        // Check that task.loc and task.loc.json versions match
+        if ((taskJson.version.Major !== taskLocJson.version.Major) || 
+            (taskJson.version.Major !== taskLocJson.version.Major) ||
```
Comment (damccorm) [inline make.js]: You're checking the major version 3 times, should be minor and patch

File: make.js (hunk 2 above)
Comment (stephenmichaelf) [reply make.js]: Fixed.

File: make.js (hunk 3 above)
Comment (stephenmichaelf) [reply make.js]: Fixed.

**Pull request merged**